autoload/SpaceVim/layers/autocomplete.vim

```diff
@@ -22,18 +22,18 @@
 
 function! SpaceVim#layers#autocomplete#plugins() abort
   let plugins = [
-        \ ['honza/vim-snippets',          { 'on_i' : 1, 'loadconf_before' : 1}],
-        \ ['Shougo/neco-syntax',          { 'on_i' : 1}],
-        \ ['ujihisa/neco-look',           { 'on_i' : 1}],
-        \ ['Shougo/context_filetype.vim', { 'on_i' : 1}],
-        \ ['Shougo/neoinclude.vim',       { 'on_i' : 1}],
+        \ ['honza/vim-snippets',          { 'on_event' : 1, 'loadconf_before' : 1}],
```

Comment:
Thanks, @wsdjeg. Let me try to figure out the right dein events. No much experience with dein and VimL.